smart_contract/executor/src/default/tokens.rs

```diff
@@ -49,6 +49,28 @@ macro_rules! declare_tokens {
             };
         }
 
+        /// Enum with every default token
+        #[allow(clippy::enum_variant_names)]
+        pub(crate) enum AnyPermissionToken {
```

Comment:
```suggestion
        #[non_exhaustive]
        #[allow(clippy::enum_variant_names)]
        pub(crate) enum AnyPermissionToken {
```

should we make it `non_exhaustive`?

Reply:
It won't help because this annotation doesn't work inside crate and this type available only within crate.

> Within the defining crate, non_exhaustive has no effect.